components/speedreader/speedreader_throttle.cc

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/speedreader/speedreader_throttle.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/no_destructor.h"
+#include "base/command_line.h"
+#include "base/strings/pattern.h"
+#include "base/strings/string_split.h"
+#include "brave/components/speedreader/speedreader_loader.h"
+#include "brave/components/speedreader/speedreader_switches.h"
+#include "mojo/public/cpp/bindings/pending_receiver.h"
+#include "mojo/public/cpp/bindings/pending_remote.h"
+#include "services/network/public/cpp/resource_response.h"
+
+namespace speedreader {
+
+namespace {
+std::vector<std::string> GetHardcodedWhitelist() {
+  static const base::NoDestructor<std::vector<std::string>> whitelist({
```

Comment:
at some point in the near future we should probably convert this whitelist into a component so we can update it more easily

Reply:
yes, I think the plan is to ship it with a good non-hardcoded whitelist